<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioPlayer</class>
 <widget class="QWidget" name="AudioPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1064</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AudioPlayer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{

background-repeat:no-repeat;



}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="pushButton_add">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background-image: url(:/rec/image/add.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{

	background-image: url(:/rec/image/add-hover.png);
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image: url(:/rec/image/add.png);
background-repeat:no-repeat;
background-position:center center;

}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
background-color: rgba(255, 255, 255,150);
border-style:outset;

border-width:0px;

border-radius:10px;
border-color: rgba(0,0,0,100);
font:bold 14px;
color:rgb(180,180,180);
padding:6px;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_search">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background-image: url(:/rec/image/sereach.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{


background-image: url(:/rec/image/sereach-hover.png);
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image: url(:/rec/image/sereach.png);
background-repeat:no-repeat;
background-position:center center;

}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton_min">
           <property name="minimumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
border-radius:0px;
background-image: url(:/rec/image/min.png);
background-color: rgba(255, 255, 255,0);
background-repeat:no-repeat;
border:none;
background-position:center center;

}

QPushButton:hover{



	background-color: rgb(255, 255, 255,100);


}
QPushButton:pressed{
	background-color: rgba(95, 95, 95,100);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);

}
</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_max">
           <property name="minimumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
border-radius:0px;
background-image: url(:/rec/image/max.png);
background-color: rgba(255, 255, 255,0);
background-repeat:no-repeat;
border:none;
background-position:center center;

}

QPushButton:hover{



	background-color: rgb(255, 255, 255,100);


}
QPushButton:pressed{
	background-color: rgba(95, 95, 95,100);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);

}
</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_quit">
           <property name="minimumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
border-radius:0px;
background-image: url(:/rec/image/quit.png);
background-color: rgba(255, 255, 255,0);
background-repeat:no-repeat;
border:none;
background-position:center center;

}

QPushButton:hover{



	background-color: rgb(255, 255, 255,100);


}
QPushButton:pressed{
	background-color: rgba(95, 95, 95,100);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);

}
</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="25,75">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_music">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
background-image: url(:/rec/image/music.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{

	background-image: url(:/rec/image/music-hover.png);
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image: url(:/rec/image/music.png);
background-repeat:no-repeat;
background-position:center center;

}
</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_like">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
background-image: url(:/rec/image/like.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{

	background-image: url(:/rec/image/like-hover.png);
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image: url(:/rec/image/like.png);
background-repeat:no-repeat;
background-position:center center;

}
</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_list">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
background-image: url(:/rec/image/list.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{

	background-image: url(:/rec/image/list-hover.png);
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image: url(:/rec/image/list.png);
background-repeat:no-repeat;
background-position:center center;

}
</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_resently">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
background-image: url(:/rec/image/play-resently.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{

	background-image: url(:/rec/image/play-resently-hover.png);
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image: url(:/rec/image/play-resently.png);
background-repeat:no-repeat;
background-position:center center;

}
</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_previous">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background-image: url(:/rec/image/pre.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{

	background-image: url(:/rec/image/pre-hover.png);
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image: url(:/rec/image/pre.png);
background-repeat:no-repeat;
background-position:center center;

}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_play">
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{

background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{


background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-repeat:no-repeat;
background-position:center center;

}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_stop">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-image: url(:/rec/image/stop.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{

	background-image: url(:/rec/image/stop-hover.png);
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image: url(:/rec/image/stop.png);
background-repeat:no-repeat;
background-position:center center;

}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_next">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background-image: url(:/rec/image/next.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{

	background-image: url(:/rec/image/next-hover.png);
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image: url(:/rec/image/next.png);
background-repeat:no-repeat;
background-position:center center;

}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="styleSheet">
        <string notr="true">
    QSlider::add-page:Horizontal
     {     
background-color: rgba(255, 255, 255,100);
        height:4px;  
     } 
     QSlider::sub-page:Horizontal   
    {  
background-color: rgb(255, 255, 255);
        height:4px;
     }
    QSlider::groove:Horizontal   
    { 
        background:transparent; 
        height:4px;
    }  
    QSlider::handle:Horizontal  
    { 
        height:16px; 
        width:16px; 
	
	border-image: url(:/rec/image/circle.png);
  margin: -6 0px; 
    }
      
</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_nowtime">
       <property name="text">
        <string>00:00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_time">
       <property name="text">
        <string>00:00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_loop">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>24</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-repeat:no-repeat;
background-position:center center;

}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ilike">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-repeat:no-repeat;
background-position:center center;

}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_position">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background-image: url(:/rec/image/position.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{
background-image: url(:/rec/image/position-hover.png);
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image: url(:/rec/image/position.png);
background-repeat:no-repeat;
background-position:center center;

}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_playlist">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background-image: url(:/rec/image/playlist.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{
background-image: url(:/rec/image/playlist-hover.png);
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image: url(:/rec/image/playlist.png);
background-repeat:no-repeat;
background-position:center center;

}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_voice">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{

background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{


background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{

background-repeat:no-repeat;
background-position:center center;

}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
